tests/test_decorator.py

```diff
@@ -80,7 +81,8 @@ def test_pep614():
     assert not wrap(("(any, expression, we, like)",), "no match here")
 
 
-common_flags = ["--select=TRIO", "tests/trio_options.py"]
+file_path = str(Path(__file__).parent / "trio_options.py")
+common_flags = ["--select=TRIO", file_path]
```

Comment:
File path is now no longer hardcoded, but expects `trio_options.py` in the same directory. Not strictly necessary since this directory isn't moved, but makes it more robust.